src/blocks/testimonials/block.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "coblocks/testimonials",
+	"category": "layout",
+	"attributes": {
```

Comment:
We need to make new blocks using the v2 blocks API. Here is an example block.json file. https://github.com/godaddy-wordpress/coblocks/blob/master/src/blocks/counter/block.json

Specifically, we need to use the version declaration but we also can then use block supports. Another comment in this file regarding fontSize.